packages/itwin/ec3-widget/src/index.d.ts

```diff
@@ -0,0 +1,5 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+declare module "*.png";
```

Comment:
What does this do?

Reply:
Most likely a leftover from the login modal which had a png. Deleted this file.